modules/web-console/web-console-server/src/main/java/org/apache/ignite/console/web/socket/WebSocketsManager.java

```diff
@@ -80,11 +93,13 @@
     /**
      * Default constructor.
      */
-    public WebSocketsManager() {
+    public WebSocketsManager(Ignite ignite) {
         agents = new ConcurrentLinkedHashMap<>();
         browsers = new ConcurrentHashMap<>();
         clusters = new ConcurrentHashMap<>();
         requests = new ConcurrentHashMap<>();
+
+        new IgniteThread(new ClusterTimeoutWorker(ignite.name(), "session-timeout-worker", log)).start();
```

Comment:
I do not like this. How about to use Spring "@Scheduled" (or something like this) instead?